packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -244,8 +284,12 @@ class AnimationController extends Animation<double>
 
   /// Starts running this animation in reverse (towards the beginning).
   ///
-  /// Returns a [Future] that completes when the animation is complete.
-  Future<Null> reverse({ double from }) {
+  /// Returns a [TickerFuture] that completes when the animation is complete.
```

Comment:
Yeah, it's confusing. I changed this to `Returns a [TickerFuture] that completes when the animation is dismissed.` which is still confusing but at least it's consistent.